client/src/templates/Challenges/components/hotkeys.tsx

```diff
@@ -61,24 +61,27 @@ const keyMap = {
   showShortcuts: 'shift+/'
 };
 
-interface HotkeysProps
-  extends Pick<ChallengeMeta, 'nextChallengePath' | 'prevChallengePath'> {
+type HotkeysProps =
+ Pick<ChallengeMeta, 'nextChallengePath' | 'prevChallengePath'> & Pick<
```

Comment:
```suggestion
type HotkeysProps = Pick<
  ChallengeMeta,
  'nextChallengePath' | 'prevChallengePath'
> &
  Pick<
```

Gotta please the linter overlords.